src/Compilers/Core/Portable/Compilation/SymbolInfo.cs

```diff
@@ -58,7 +58,8 @@ internal ImmutableArray<ISymbol> GetAllSymbols()
         /// </summary>
         public CandidateReason CandidateReason { get; }
 
-        internal SymbolInfo(ISymbol symbol)
+        // TODO2 confirm whether this is the right change, or we should change GetMethodGroupItemsAndSelectionAsync to take an ISymbol instead
```

Comment:
I don't think this is the right change.

Reply:
I suspected so, but wanted to unblock most of the PR. I'll need some help from IDE folks to update `GetMethodGroupItemsAndSelectionAsync` since it affects an interface used for Pythia. I'm not sure how to do that safely.

Reply:
so a SymbolInfo is just data.  it's not clear to me why someone else could not cosntruct it.